frontend/.env

```
@@ -1 +1 @@
-REACT_APP_STRIPE_PUBLIC_KEY=pk_test_51SyF74H6Zvt2fDNN9h4AXvayA8r61KRnVZOKLunp3gawkDLcIa1Xg7gsD527OUtGNbcribDnWN0folZSwsFpACep00N8MFllU6
\ No newline at end of file
+REACT_APP_STRIPE_PUBLIC_KEY=pk_test_51SNm1M2Uk0012hZxopl8nlyTWVTWoAizySju16bzuQOIA7aK1VTyfHDqCuRawUG4VXsEjAQOq4T6XWktcYXsu5rc00ID5PAa17
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)